package.json

```diff
@@ -35,6 +35,7 @@
     "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 8000 --ui bdd --exit --color true",
     "test:expression": "nyc --reporter=text mocha --recursive 'test/unit/expressions/*.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
     "test:multientity": "nyc --reporter=text mocha --recursive 'test/unit/ngsiv2/plugins/multientity-plugin_test.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
+    "test:JEXL": "nyc --reporter=text mocha --recursive 'test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js' --reporter spec --timeout 5000 --ui bdd --exit --color true",
```

Comment:
Should this be included at the end? Or it was just for debugging?

In the former case, I'd suggest to use `test:jexl` (style)

Reply:
debugging I will removeit

Reply:
solved